### PR TITLE
Explain 行 and 列

### LINKED ISSUE #369: Translation issue and question about "Arrangement of tables and/or illustrations"
In 2.4.2 (3)(1 and 2), the translation for the sub points are weird and labelled "Check this translation". However, there is a bigger confusion arises from these line.

https://github.com/w3c/clreq/blob/d4cf48684f03e75956a7459be04b3139c32e0e9b/index.html#L1226
https://github.com/w3c/clreq/blob/d4cf48684f03e75956a7459be04b3139c32e0e9b/index.html#L1231

In Chinese, there are two terms that can be used to describe a table: 行 and 列. The corresponding terms in English are "row" and "column". However, in different regions, the Chinese terms are used differently: in China, 行 refers to row and 列 refers to column; in Taiwan however, 列 refers to row and 行 refers to column. [(Reference)](https://zh.wikipedia.org/wiki/Topic:U3b8ags204w4rqvy) This might causes issues when trying to describe how tables are arranged across different regions. My understanding is that 行 usually goes in the same direction as the general writing direction, while 列 is perpendicular to the direction of writing. For the ease of understanding, I have made this following diagram:

image

This will involve how the terms in English should be mapped: either the row/header are described in terms of relative writing direction, or in terms of absolute direction. I would prefer the terms in English to be described in terms of relative writing direction, thus 列 will always be translated to column, while 行 will always be translated to row with no regards to the writing direction. Why? If absolute directions are used (i.e. row = horizontal, column = vertical), the direction of increment for row (horizontal) will be in the opposite direction when in vertical writing mode (列 goes from right to left).

Also, the section title was "Arrangement of tables and/or illustrations."/"表格、图片等标题栏的位置" which we should expect any descriptions about the title itself.

Thus, my suggestions for the translation fixes are as follow:

https://github.com/w3c/clreq/blob/d4cf48684f03e75956a7459be04b3139c32e0e9b/index.html#L1226
> In vertical writing mode, list the column (incrementing towards the left) headers at the right of table and the row (incrementing going down) headers at the top of table; put the title of the tables and/or illustrations at the left or right side.
> 直排时，表格横列（往左递增）的标题列位右侧，直行（往下递增）的标题列位上方；表格、图片等标题置放于左或右侧。

https://github.com/w3c/clreq/blob/d4cf48684f03e75956a7459be04b3139c32e0e9b/index.html#L1231
> In horizontal writing mode, list the column (incrementing going down) headers at the right of table and the row (incrementing towards the right) headers at the top of table; put the title of the tables and/or illustrations at the top or bottom.
> 横排时，表格直列（往下递增）的标题列位上方，横行（往右递增）的标题列位左侧；表格、图片等标题置放于上或下方。

### PR DESCRIPTION
**Pull request by xfq**
Fix #369.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/clreq/pull/400.html" title="Last updated on Oct 16, 2021, 4:16 AM UTC (a97cc8e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/clreq/400/7a7ca28...a97cc8e.html" title="Last updated on Oct 16, 2021, 4:16 AM UTC (a97cc8e)">Diff</a>

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/clreq/pull/400.html" title="Last updated on Jan 24, 2022, 12:35 PM UTC (ba561d8)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/clreq/400/7a7ca28...ba561d8.html" title="Last updated on Jan 24, 2022, 12:35 PM UTC (ba561d8)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1222,15 +1222,30 @@ <h4>
           <p its-locale-filter-list="zh-hans" lang="zh-hans">表格、图片等标题栏的位置</p>
           <p its-locale-filter-list="zh-hant" lang="zh-hant">表格、圖片等標題列的位置</p>
           <ol>
+            <li id="hang-lie">
+              <p its-locale-filter-list="en" lang="en">
+                In Mainland China, the direction of row in a table is called 行 which is horizontal, while the direction of column is called 列 which is vertical. In Taiwan and Hong Kong, the direction of row in a table is called 列 which is horizontal, while the direction of column is called 行 which is vertical. The inline direction (either in horizontal mode or vertical mode) is historically called as 行, with the block direction is called 列. However, these terms are not swapped in modern usage when changing writing directions out of habits. The following subpoints will indicate the incrementing direction in table to prevent confusion.
```
Comment (r12a) [inline index.html]: How about:
In Mainland China, the direction of horizontal rows in a table is called 行 (xíng), and the direction of vertical columns is called 列 (liè).  In Taiwan and Hong Kong, the direction of horizontal rows in a table is called  列 (liè), and the direction of vertical columns is called 行 (xíng). Inline direction (in either horizontal or vertical mode) is historically called 行 (xíng), and block progression direction is called 列 (liè). However, in modern usage, these terms are habitually not swapped when changing the writing direction. To help avoid confusion, the following numbered list indicates the incremental direction in tables.

Comment (NightFurySL2001) [reply index.html]: > To help avoid confusion, the following numbered list indicates the incremental direction in tables.

Should be:
To help avoid confusion, the following numbered list will indicate the row/column incremental direction when mentioned in tables.

Additional notes for Chinese:
下面的分节将会标示表格内 **行列** 递增的方向以避免混淆。
下面的分節將會標示表格內 **行列** 遞增的方向以避免混淆。

P/S: 行 (xíng) should be 行 (háng/ㄏㄤˊ) in this context.

Edit:
> However, in modern usage, these terms are habitually not swapped when changing the writing direction.

I think a better term can be: "However, in modern usage, these terms are habitually not ***exchangable*** when changing the writing direction", but it doesn't seem to work that well...

Comment (r12a) [reply index.html]: Having just read the original issue on which this PR is based (https://github.com/w3c/clreq/issues/369), i think i understand this text better, and i think the current text is confusing at best, and misleading at worst.

The text currently proposed for this PR implies that there is a difference in the use of the terms 行 and 列 base on the location of usage, but in fact there isn't. There is only a difference in the arrangement of the text in the table.  行 can be described as the inline progression direction, using CSS terminology.  列 is simply the block progression direction.

Instead of saying "In Mainland China..."  and "In Taiwan and Hong Kong...", it should instead say "In horizontally-set text..." and "In vertically-set text..." (horizontal/vertical writing mode, if you prefer).

So here is what i think is a better version of the text i suggested above:

> The direction of the rows in a table follows the inline progression direction for the text, called 行 (háng). The direction of the columns follows the block progression direction, called 列 (liè). In horizontally-set text (such as that usually used in Mainland China), the row direction (行) is left-to-right, whereas in vertically-set text (such as that used in Taiwan or Hong Kong), the row direction is top-to-bottom. The column direction (列) is top-to-bottom in horizontal text, but right-to-left in vertically-set text.

Comment (realfish) [reply index.html]: According to [Bobby’s comment](https://github.com/w3c/clreq/issues/369#issuecomment-896110913) under #369, in TC (esp. in Taiwan) nowadays, a visually vertical line in table is called 行 (háng) or 欄 (lán) while a visually horizontal line is called 列 (liè).

It means that *no matter in vertical mode or horizontal mode*, in Taiwan nowadays, the column (visually vertical) is called 直**行**/直欄 and the row (visually horizontal) is called 橫**列**. The Taiwanese idiom “直行横列” literally means “vertical *háng* and horizontal *liè*”. However in Mainland China nowadays, it is quite the reverse.

In short, Taiwan (TC) translates “column” into “行” and “row” into “列”, while Mainland (SC) translates “column” into “列” and “row” into “行” nowadays, regardless of writing modes.

If my understanding is correct, this PR is exactly to clarify the terminological difference between TC and SC.

Comment (NightFurySL2001) [reply index.html]: > The text currently proposed for this PR implies that there is a difference in the use of the terms 行 and 列 base on the location of usage,

Yes, nowaday it is based on location of usage. Historically it should've been not the case, but the terminologies have been stuck per region now. 

I think this sentence of the original PR:
> Inline direction (in either horizontal or vertical mode) is historically called 行 (háng), and block progression direction is called 列 (liè). However.....

can be restructured to:
> Historically, the inline progression direction (in either horizontal or vertical mode) is called 行 (háng), and the block progression direction is called 列 (liè). However......

This would show the historic usage of the terms and indicate how they end up as their current status of regional use.

**Comment by xfq** [comment]
I updated the PR per https://github.com/w3c/clreq/issues/369#issuecomment-997366347 . Please review.

File: index.html [hunk 2]
```diff
@@ -1218,37 +1218,67 @@ <h4>
...
+                  </p>
+                  <p its-locale-filter-list="zh-hans" lang="zh-hans">在不同的地区，中文术语中的「行」（háng）和「列」（liè）所描述的单元格排列方向，可能是不同的。例如，中国大陆的「行」通常指一组水平排列的单元格，「列」指一组垂直排列的单元格；而台湾、香港地区的「行」通常指一组垂直排列的单元格，「列」（或「栏」）指一组水平排列的单元格。</p>
+                  <p its-locale-filter-list="zh-hant" lang="zh-hant" class="checkme">在不同的地區，中文術語中的「行」（háng）和「列」（liè）所描述的單元格排列方向，可能是不同的。例如，中國大陸的「行」通常指一組水平排列的單元格，「列」指一組垂直排列的單元格；而臺灣、香港地區的「行」通常指一組垂直排列的單元格，「列」（或「欄」）指一組水平排列的單元格。</p>
+
+                  <p its-locale-filter-list="en" lang="en" class="checkme">
+                    This document uses row/column in logical directions consistently, i.e., "row" is vertical in vertical writing mode and horizontal in horizontal writing mode, and "column" is horizontal in vertical writing mode and vertical in horizontal writing mode.
+                  </p>
+                  <p its-locale-filter-list="zh-hans" lang="zh-hans">本文档采用「逻辑行」「逻辑列」来统一表述，也即，「行」在直排模式下垂直、在横排模式下水平，「列」在直排模式下水平、在横排模式下垂直。</p>
```
Comment (NightFurySL2001) [inline index.html:1277]: Do we need to indicate the incrementing direction in here: e.g. 
> This document uses row/column in logical directions consistently, i.e., "row" is vertical going top-down in vertical writing mode and horizontal going left-right in horizontal writing mode, and "column" is horizontal going right-left in vertical writing mode and vertical going top-down in horizontal writing mode.
> 本文档采用「逻辑行」「逻辑列」来统一表述，也即，「行」在直排模式下上至下垂直、在横排模式下左至右水平，「列」在直排模式下右至左水平、在横排模式下上至下垂直。

The main reason is this edit did not reference back to W3C as suggested in https://github.com/w3c/clreq/issues/369#issuecomment-997433306, so the incrementing direction is not referenced. 
> CLReq 可以进一步考虑，是否需要对逻辑方向做出更详细的阐释。

If it is deemed ignorable then do ignore this comment.

**Comment by xfq** [comment]
Based on discussions at the [editors' call](https://www.w3.org/2022/03/04-clreq-minutes.html#t09), we decided to merge this PR.
